harness/harness-common.rb

```diff
@@ -52,10 +52,11 @@ def get_rss
 yb_env_var = ENV.fetch("RESULT_JSON_PATH", default_path)
 YB_OUTPUT_FILE = File.expand_path yb_env_var
 
-def return_results(times)
+def return_results(warmup_iterations, counted_iterations)
   yjit_bench_results = {
     "RUBY_DESCRIPTION" => RUBY_DESCRIPTION,
-    "values" => times,
+    "warmup" => warmup_iterations,
+    "counted" => counted_iterations,
```

Comment:
I think we call it "bench" or "benchmark iterations", e.g. `MIN_BENCH_ITRS` and its description in readme. They should be called consistently. If you want to rename it, please consider renaming the environment variable as well.

Reply:
I agree bench_iterations would be a better name.

Reply:
Done.